javascript.js

```diff
@@ -12,3 +12,157 @@ hamburger.addEventListener('click', display);
 for (let i = 0; i < mobileItem.length; i += 1) {
   mobileItem[i].addEventListener('click', display);
 }
+
+const project = {
```

Comment:
Please be sure that the JavaScript code implements an array of objects to store JavaScript objects containing the information for each project. This way you will have the proper architecture to easily access the information for multiple projects in the near future and meet the project requirements 💪😎

    const projects = [{
        name: "Submarine game",
        description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
        image: "link_to_image.jpg",
        technologies: ["Phaser3", "JavaScript", "Jest"],
        link: "link_to_live_version.com",
    },
    {
         name: "Submarine game 2",
         description: "Other description",
         image: "link_to_image.jpg",
         technologies: ["Phaser5", "JavaScrip2t", "Test"],
         link: "link_to_live_version_2.com",
     }]